src/injected/shadow-initializer.ts

```diff
@@ -38,7 +38,7 @@ export class ShadowInitializer {
 
         const shadowHostElement = this.createDivWithId('insights-shadow-host');
 
-        rootContainer.appendChild(shadowHostElement);
+        rootContainer?.appendChild(shadowHostElement);
```

Comment:
This makes it so the method is a no-op if it can't do its job, where previously it would throw an error; I think throwing an error would probably be preferable if the root container can't be found